app-emulation/box64/autobuild/overrides/usr/lib/binfmt.alternatives/box64.conf

```diff
@@ -0,0 +1 @@
+:box64:M::\x7f\x45\x4c\x46\x02\x01\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\x02\x00\x3e\x00:\xff\xff\xff\xff\xff\xfe\xfe\xfc\xff\xff\xff\xff\xff\xff\xff\xff\xfe\xff\xff\xff:/usr/bin/box64:CF
```

Comment:
This makes no sense and differs from the upstream copy.

Overriding upstream-supplied files is asking for trouble - you won't notice upstream changes this way.